library/src/morpheus-master/sass/themes/_light.scss

```diff
@@ -234,15 +233,15 @@
     // Error banner
     --errorBanner-bordercolor: var(--sakai-color-red-5);
     --errorBanner-bgcolor: var(--sakai-color-red-0);
-    --errorBanner-color: var(--sakai-color-red-0);
+    --errorBanner-color: var(--sakai-color-red-8);
 
     /* Swapped 'View As' view */
-    --swapped-view-primary:            var(--sakai-color-orange-6);
```

Comment:
Whitespace is removed for the next several lines, but then returns before being removed again. Let's be consistent.